collector/model/metric.go

```diff
@@ -82,7 +83,7 @@ func (g *GaugeGroup) String() string {
 }
 
 func (g *GaugeGroup) Reset() {
-	g.Name = ""
+	g.Name = constlabels.NetWorkAnalyzeGaugeGroup
```

Comment:
Why is it reset to this name? The Reset() method should not assume any names.

Reply:
Already changed